package.json

```diff
@@ -50,6 +50,7 @@
     "webpack-node-externals": "^2.5.2"
   },
   "dependencies": {
-    "@comicrelief/lambda-wrapper": "^1.1.10"
+    "@comicrelief/lambda-wrapper": "^1.1.10",
+    "axios": "^0.26.1"
```

Comment:
> Suggesting we update to the latest version of `axios` before merging.

Latest version of axios doesn't seem to play nicely with jest :(

Reply:
https://github.com/axios/axios/issues/5101

Reply:
@corinja currently working on #73, which migrates this repo to TypeScript as per the discussion in the planning session yesterday. I've made some changes to the jest setup here so it may well be the case that it will be fine. This PR is most of the way there, and is largely just waiting on this PR to be merged.

I can port these changes over to my PR and see if there is any issue with the axios version if that is a suitable suggestion?

Reply:
> Hey @corinja, I'm currently working on #73, which migrates this repo to TypeScript as per the discussion in the planning session yesterday. I've made some changes to the jest setup here so it may well be the case that it will be fine. This PR is most of the way there, and is largely just waiting on this PR to be merged.
> 
> I can port these changes over to my PR and see if there is any issue with the axios version if that is a suitable suggestion?

I didn't see your comment before posting my update @pauldolden I'm just about to push my changes FYI

Reply:
Done!